base/utils/R/read.output.R

```diff
@@ -179,13 +179,13 @@ read.output <- function(runid, outdir, start.year = NA, end.year = NA, variables
 ##'--------------------------------------------------------------------------------------------------#
 ##' Converts the output of all model runs
 ##'
-##' @title convert outputs from model specific code to 
+##' @title convert outputs from model specific code to pecan standard (deprecated) 
```

Comment:
Let's remove this entirely rather than add another commit changing a dead function

Reply:
I agree, I was just waiting to get the approval from @mdietze or @robkooper but given it isnt touched by any part of the code its probably fine